ckanext/canada/commands.py

```diff
@@ -439,6 +445,72 @@ def changed_datasets(self, since_date):
     def rebuild_external_search(self):
         search_integration.rebuild_search_index(LocalCKAN(), self.options.unindexed_only, self.options.refresh_index)
 
+    def load_suggested(self, filename):
+        """
+        a process that loads suggested datasets from Drupal into CKAN
+        """
+        registry = LocalCKAN()
+
+        # load packages as dict
+        results = True
+        counter = 0
+        batch_size = 100
+        existing_suggestions = {}
+        while results:
+            packages = registry.action.package_search(q='type:prop', start=counter, rows=batch_size, include_private=True)['results']
+            if packages:
+                for package in packages:
+                    existing_suggestions[package['id']] = package
+                counter += batch_size
```

Comment:
use `counter += len(packages)` to handle package_search returning less than you expect (e.g. because of configuration)